src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1524,6 +1524,10 @@ private function getErrorMessages(Exception $e)
                     'Invalid action: this order has not been paid.',
                     'Admin.Notifications.Error'
                 ),
+                InvalidOrderStateException::NOT_FOUND => $this->trans(
+                    'Invalid order state id.',
```

Comment:
@MatShir Need wording

Reply:
Ping @LouiseBonnard

Reply:
Same here. :-)

Reply:
It's an error in an alert box if the form is submitted and order state is not right filled.

Reply:
Order state or order status? I suggest _You must choose an order status to create the order._ in _Admin.Orderscustomers.Notification_.